contracts/IICO.sol

```diff
@@ -2,7 +2,7 @@
  *  @author Clément Lesaege - <clement@lesaege.com>
  */
 
-pragma solidity ^0.4.18;
+pragma solidity 0.4.23; //Stable version to 29/04/2018
```

Comment:
Did you manage to make truffle work with the last version?

Reply:
Ok, there has been a new truffle release, so now we can update to 0.4.23

Reply:
There is no warranty that this contract will compile on the future if there are heavy compiler changes, thats why it matter (once you are on production) to set a fixed compiler version (right now doesn't matter if it's variable since it's on revision)